Angular13/Angulardemo/src/app/app.module.ts

```diff
@@ -14,8 +14,13 @@ import { initializeApp,provideFirebaseApp } from '@angular/fire/app';
 import { environment } from '../environments/environment';
 import { provideAuth,getAuth } from '@angular/fire/auth';
 import { provideFirestore,getFirestore } from '@angular/fire/firestore';
+<<<<<<< Updated upstream
 import { HttpClientModule } from '@angular/common/http';
+import { FlexLayoutModule } from "@angular/flex-layout";
 
+=======
+import { FlexLayoutModule } from "@angular/flex-layout";
+>>>>>>> Stashed changes
```

Comment:
can you remove this? import same 2 times. also please remove line -17,21,23....in your next PR